src/elements/content-sidebar/Sidebar.js

```diff
@@ -127,6 +127,13 @@ class Sidebar extends React.Component<Props, State> {
             this.setState({ isDirty: true });
         }
 
+        if (boxAISidebarProps && boxAISidebarProps.shouldOpenBoxAISidebar) {
+            // navigate to boxai tab and open sidebar
+            history.push(`/${SIDEBAR_VIEW_BOXAI}`);
+            this.isForced(true);
```

Comment:
Can we use the existing [location state](https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/Sidebar.js#L241) option for this behavior?